forge/test/models/pytorch/text/llama/test_llama3.py

```diff
@@ -11,7 +11,14 @@
 import forge
 from transformers.models.llama.modeling_llama import LlamaModel, Cache, StaticCache, AttentionMaskConverter
 
-variants = ["meta-llama/Meta-Llama-3-8B", "meta-llama/Meta-Llama-3-8B-Instruct"]
+variants = [
+    "meta-llama/Meta-Llama-3-8B",
+    "meta-llama/Meta-Llama-3-8B-Instruct",
+    "meta-llama/Llama-3.1-8B",
+    "meta-llama/Meta-Llama-3.1-8B-Instruct",
```

Comment:
can you change `meta-llama/Meta-Llama-3.1-8B-Instruct` to `meta-llama/Llama-3.1-8B-Instruct` as version you listed doesn't exist.

Reply:
Sure will modify in next patch